articles/scopes/preview/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Scopes
+description: Overview of scopes.
+toc: true
+---
+
+# Scopes
+
+<div class="alert alert-info">
+<strong>Heads up!</strong> This article describes the latest Auth0 implementation, according to the <a href="/api-auth">API Authorization flows</a>. If you are looking for the legacy Scopes doc refer to <a href="/scopes">Scopes</a>.
+</div>
+
+In OpenID Connect (OIDC) we have the notion of __scopes__, or __claims__. These are strings, sent as part of the `scope` request parameter. These claims can be standard (as defined by the specification) or custom.
+
+
+Scopes are used in the following cases:
+
+- When you want to get an `id_token` in the response. In this case, you need to include `openid` as part of the `scope` request parameter: `scope=openid`.
```

Comment:
I'm not sure this is true. Including `scope:openid` will add `userinfo` as audience to the access token.
If users want to get an `id_token` they should use `response_type:id_token`